bin/drop_filter_results.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+import argparse
+import re
+from collections import OrderedDict
+from pathlib import Path
+import csv
+import os
+import pandas as pd
+import pyreadr
+
+
+def FilteringResults(sample, gene_panel, out_drop_ae_rds, out_drop_gene_name, out_drop_as_tsv):
```

Comment:
```suggestion
def FilteringResults(samples: list, gene_panel: str, out_drop_ae_rds: str, out_drop_gene_name: str, out_drop_as_tsv: str):
```
Makes it clearer that this is a list of samples. Also type hints